File: js/problem-1008.js
// https://www.beecrowd.com.br/judge/en/problems/view/1008

let input = require('fs').readFileSync('/dev/stdin', 'utf8');
let [number, workedHours, receivedPerHour] = input.split('\n').map(value => Number(value));
let salary = workedHours * receivedPerHour;
console.log(`NUMBER = ${number}`);
console.log(`SALARY = U$ ${salary.toFixed(2)}`);

File: python/problem-1008.py
# https://www.beecrowd.com.br/judge/en/problems/view/1008

number = int(input())
worked_hours = int(input())
receveid_per_hour = float(input())
salary = worked_hours * receveid_per_hour
print(f"NUMBER = {number}")
print(f"SALARY = U$ {salary:.2f}")